src/alda/repl/commands/ReplInfo.java

```diff
@@ -0,0 +1,103 @@
+package alda.repl.commands;
+
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.JsonParseException;
+
+import alda.AldaResponse.AldaScore;
+import alda.AldaServer;
+import alda.error.NoResponseException;
+import alda.error.ParseError;
+import jline.console.ConsoleReader;
+
+public class ReplInfo implements ReplCommand {
+  
+  private Gson gson = new Gson();
+	
+  @Override
+  public void act(String args, StringBuffer history, AldaServer server,
+                  ConsoleReader reader, Consumer<AldaScore> newInstrument)
+                		  throws NoResponseException {
+	try {
+	  String res = server.parseRaw(history.toString(), "data");
+	  if(res == null) {
+	    System.err.println("An internal error occurred when reading the score.");
+	  } else {
+        JsonObject scoreMap = gson.fromJson(res, JsonObject.class);
+  
+        StringBuilder sb = new StringBuilder();
+        sb.append("Instruments: ")
+        	.append(getInstrumentsString(scoreMap))
+        	.append(System.lineSeparator());
+        
+        sb.append("Current instruments: ")
+    		.append(getCurrentInstrumentsString(scoreMap))
+    		.append(System.lineSeparator());
+        
+        sb.append("Events: ")
+    		.append(getEventsAmount(scoreMap))
+    		.append(System.lineSeparator());
+     
+        sb.append("Markers: ")
+    		.append(getMarkersAmount(scoreMap))
+    		.append(System.lineSeparator());
+        
+		System.out.println(sb);
+	  }
+	  
+	} catch(ParseError e) {
```

Comment:
Since these clases are all the same catch block, I think you can:

```
catch(ParseError | JsonParseException | ClassCastException e) {
...
}
```

Why are you catching a classcastexception, by the way? I hope that isn't coming up here :P

Reply:
I've noticed that in the returned JSON "markers" and "instruments" are objects, while "current-instruments" and "events" are lists. Not sure why they are not all lists. This gave me a certain degree of uncertainty on the JSON structure contract reliability.
In case the contract changes in the future (i.e., one of such objects is changed to a list), a ClassCastException would be thrown.
But this was just due to some (wrong?) impression of mine, so if you say that the JSON structure is intended to be kept as is, I can remove it.

Reply:
That's a good concern to have. I think we can reasonably rely on the shape of the data not changing, in this case. If we ever do want to change it for some reason, we'll just have to remember to adjust client code like this at the same time.

Reply:
Fixed.